src/main/java/gov/cms/madie/models/measure/Reference.java

```diff
@@ -5,12 +5,22 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
 
 @Data
 @SuperBuilder(toBuilder = true)
 @NoArgsConstructor
 public class Reference {
 	@Id private String id;
 	private String referenceText;
+	@NotNull(
```

Comment:
Do we need separate @NotNull and and @NotEmpty since there is already @NotBlank?

Reply:
You're right, removed